GOVERNANCE.md

```diff
@@ -0,0 +1,170 @@
+
+### Master branch
+
+The master branch must be kept in a releasable state.
+
+Trivial changes may be committed directly to a repository. Changes deemed trivial are things like small documentation typos, ecosystem changes (eg, CI configuration, linting rules), etc. Obviously, *trivial* is subjective and anyone is welcome to comment directly on such changes or open an issue regarding them.
+
+All other changes must be reviewed by the committers. This includes contributions from the community (contributors) as well as the current committers. We use [Github pull requests](https://help.github.com/articles/about-pull-requests/) and [Github pull request reviews](https://help.github.com/articles/about-pull-request-reviews/) to manage reviewing and merging the proposed changes.
+
+For small changes, at least one committer must approve the change. If a committer was the author of the change being reviewed, the approval must come from a different committer. For large, significant, or breaking changes, the committers must reach consensus on the change. Like *trivial*, the notion of *small* and *large* are subjective terms. However, a breaking change should be clear and unambiguous. Note that the size of the contribution does not determine whether the change requires just one approval or consensus. Rather, the impact of the change reflects the type of review required. For instance, the addition of a single optional field to the API described in the specification would be considered a significant change and would warrant consensus amongst the committers.
+
+Changes to the specification are scrutinized far more closely than that of documentation or related code projects (like sample projects, test harnesses, etc). As such, specification changes are always considered significant and require consensus from the committers.
+
+### Feature branches
+
+Committers should use feature branches to work on in-progress features and changes. Committers may commit directly to feature branches at any time and without prior review.
+
+Committers can submit a pull request from their feature branch to the master branch for review.
+
+### Forks
+
+As contributors do not have write access to the repository, their contributions must be done from personal forks.
+
+Contributors can submit a pull request from their fork to the upstream master branch for review.
+
+## Voting and Consensus
+
+Anyone is welcome to comment on proposed changes. This includes expressing their opinion on why a proposed change should or should not be approved. Ultimately, committers make the final determination on accepting a proposed change.
+
+Committers should seek out the opinions and experiences from the broader community. It is the committers responsibility to balance the feedback from the community along with their own opinions and experiences.
+
+Committers must express their votes as:
+
+### :+1: Approve
```

Comment:
Out of curiosity, if we're using the "review changes" functionality of the github PR, do we also need to do the +1/-1? Or is approving/request changes good for the +1/-1?

I can see that abstaining, if it needs to be a more solid "op out" than not doing a review, may require a more formal comment.

thoughts?

Reply:
Hey @daliboz - nice, careful read!

I updated to better clarify that the formal github actions on a PR are equivalent to 👍 or 👎